pypesto/petab/importer.py

```diff
@@ -130,29 +132,43 @@ def check_gradients(
             modes = [mode]
 
         if multi_eps is None:
-            multi_eps = np.array([10**(-i) for i in range(3, 9)])
+            multi_eps = np.array([10 ** (-i) for i in range(3, 9)])
 
         for mode in modes:
             try:
-                dfs.append(objective.check_grad_multi_eps(
-                            free_indices, *args, **kwargs,
-                            mode=mode, multi_eps=multi_eps))
+                dfs.append(
+                    objective.check_grad_multi_eps(
+                        free_indices,
+                        *args,
+                        **kwargs,
+                        mode=mode,
+                        multi_eps=multi_eps,
+                    )
+                )
             except (RuntimeError, ValueError):
                 # Might happen in case PEtab problem not well defined or
                 # fails for specified tolerances in forward sensitivities
                 return False
 
-        return all([
-            any([
-                np.all((mode_df.rel_err.values < rtol) |
-                       (mode_df.abs_err.values < atol)),
-            ])
-            for mode_df in dfs
-        ])
-
-    def create_model(self,
-                     force_compile: bool = False,
-                     **kwargs) -> 'amici.Model':
+        return all(
+            [
+                any(
+                    [
+                        np.all(
+                            (mode_df.rel_err.values < rtol)
+                            | (mode_df.abs_err.values < atol)
+                        ),
+                    ]
+                )
+                for mode_df in dfs
+            ]
+        )
```

Comment:
:sweat_smile:

Reply:
Agree with Daniel here, is there a way to tell black that we want `all([` and `any([`?